packages/contracts/contracts/engine/V3/forks/PROHIBITION/AdminACLV0_PROHIBITION.sol

```diff
@@ -200,14 +200,29 @@ contract AdminACLV0_PROHIBITION is IAdminACLV0_PROHIBITION, ERC165 {
      * @param _contract The contract address.
      * @param _selector The function selector.
      * @return isApproved The approval status.
-     * @dev this function is public insteaad of internal so that the right to toggle approvals
+     * @dev this function is public instead of internal so that the right to toggle approvals
      * can also be delegated
      */
     function allowed(
         address _sender,
         address _contract,
         bytes4 _selector
     ) public view returns (bool) {
+        IGenArt721CoreContractV3_Base coreV3 = IGenArt721CoreContractV3_Base(
+            _contract
+        );
+
+        bytes4 addProjectSelector = bytes4(
+            keccak256(bytes("addProject(string,address)"))
+        );
+
+        if (_selector == addProjectSelector && coreV3.nextProjectId() <= 10) {
```

Comment:
Nit: Can you add a dev comment please explaining why you use this magic number of 10?

Reply:
will do!

Reply:
Reverted this. The Admin can choose which contracts/EOAs can get through as it is.
No need to add gas to every single ACL check

Reply:
resolving this based on latest strategy no longer using magic number of 10 👍